content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -294,6 +294,8 @@ command line arguments to `kube-apiserver`:
 * `--service-account-signing-key-file`
 * `--api-audiences` (can be omitted)
 
+To enable non-disruptive change of issuer we can specify `--service-account-issuer` flag multiple times to `kube-apiserver` for k8s v1.22 and later versions.
+
 {{< /note >}}
```

Comment:
```suggestion
{{< /note >}}

You can specify the `--service-account-issuer` argument multiple times. This can be useful to
enable a non-disruptive change of the key for an issuer.
You must be running running Kubernetes v1.22 or later to be able to specify
`--service-account-issuer` multiple times`.
```

_Is that true?_ How does that work if there are two different keys / signing keys?

Reply:
Hi, @sftim thanks for looking at this. I will modify this according to your suggestion soon.

> _Is that true?_ How does that work if there are two different keys / signing keys?

Yes, it is true.  It was merged in k8s v1.22. See this [PR](https://github.com/kubernetes/kubernetes/pull/101155)  for more information.

Reply:
Hi @sftim 
I have Addressed the comment, please take a look.

Thanks